lib/fennec/evaluator.ex

```diff
@@ -2,16 +2,15 @@ defmodule Fennec.Evaluator do
   @moduledoc false
 
   alias Jerboa.Params
+  alias Fennec.TURN
 
-  @spec service(Params.t, map) :: Params.t | :void
-  def service(p, changes) do
+  @spec service(Params.t, map, TURN.t) :: {Params.t, TURN.t} | :void
+  def service(p, changes, turn_state) do
```

Comment:
Doesn't that belong to the TURN state anyway? For me it looks artificial to have `changes` to store ip and port.